.github/workflows/tests.yml

```
@@ -68,12 +68,13 @@ jobs:
           restore-keys: ${{ runner.os }}-go-
 
       - name: Setup Artifactory
-        run: |
-          go install github.com/jfrog/jfrog-testing-infra/local-rt-setup@latest
-          ~/go/bin/local-rt-setup
         env:
+          setup-url: github.com/jfrog/jfrog-testing-infra/local-rt-setup@latest
           RTLIC: ${{secrets.RTLIC}}
           GOPROXY: direct
+        run: |
+          go install ${{env.setup-url}} || go install ${{env.setup-url}}
```

Could you please try this?

```suggestion
          for i in {1..3}; do go install github.com/jfrog/jfrog-testing-infra/local-rt-setup@latest && break; done
```

Explanation
The following command is the core of the loop: `go install github.com/jfrog/jfrog-testing-infra/local-rt-setup@latest && break`
If go install doesn't fail, do break.